pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -490,6 +611,18 @@ func publishAttestNonce(ctx *attestContext) {
 	log.Debugf("[ATTEST] publishAttestNonce done for %s", key)
 }
 
+func publishStorageKeyFromController(ctx *attestContext, storageKey []byte) {
```

Comment:
Name of function vs name of publication makes this confusing.

Reply:
Corrected it